reference/src/representation/integers-floatingpoint.md

```diff
@@ -0,0 +1,47 @@
+# Representation of Boolean, Floating Point, and Integral Types
+This chapter represents the consensus from issue [#9]. It documents the memory layout and considerations for `bool`, `usize`, `isize`, floating point types, and integral types.
+[#9]: https://github.com/rust-rfcs/unsafe-code-guidelines/issues/9
+
+## Overview
+These are all scalar types, representing a single value. These types have no representation variants (no #[repr(C)] or #[repr(Rust)]. Their size is fixed and well-defined across FFI boundaries and map to their corresponding integral types in the C ABI.
+- `bool`: 1 byte
```

Comment:
I think we do want to guarantee here that it has the same layout as `_Bool` as well as a size of 1 byte.